src/main/cpp/hw/DragonSparkFlex.cpp

```diff
@@ -198,39 +189,34 @@ void DragonSparkFlex::SetVoltage(units::volt_t output)
 
 bool DragonSparkFlex::IsMotorInverted() const
 {
+    return m_spark->GetInverted();
 }
 
 bool DragonSparkFlex::IsForwardLimitSwitchClosed()
 {
-    return false;
+    return m_forwardLimitSwitch.Get();
 }
 
 bool DragonSparkFlex::IsReverseLimitSwitchClosed()
 {
-    return false;
+    return m_reverseLimitSwitch.Get();
 }
 
-void DragonSparkFlex::EnableDisableLimitSwitches(
-    bool enable)
+void DragonSparkFlex::EnableDisableLimitSwitches(bool enable)
 {
+    SparkLimitSwitch limitSwitch = m_spark->GetForwardLimitSwitch(m_forwardType);
```

Comment:
use member variables for the limit switches. Both switches need to be enabled or disabled